go.mod

```diff
@@ -23,5 +23,6 @@ require (
 	k8s.io/apimachinery v0.20.2
 	k8s.io/client-go v0.20.2
 	k8s.io/klog v1.0.0
+	k8s.io/kubectl v0.20.2
```

Comment:
@Ab-hishek is this dependency really needed? Looks like we are adding a lot of unnecessary stuff to get one small thing done.

Reply:
That's a valid point. I'll try to remove this dependency.